src/components/Canvas/CanvasTemplate.vue

```diff
@@ -206,18 +227,20 @@ export default {
           imageToAnalyzeTopPositionWithPixel.length - 2
         );
         const imageId = imageToAnalyze.imageId;
+        const tag = imageToAnalyze.tag;
         // Check collision
         this.collisionAnalysis(
           imageId,
           {
             top: imageToAnalyzeImageTopPosition,
             left: imageToAnalyzeImageLeftPosition,
           },
-          focusElement
+          focusElement,
+          tag
```

Comment:
probably makes more sense to send imageToAnalyze instead of 2 paramters imageId and tag

Reply:
yes maybe, i will refactor this part

Reply:
refactoring done